plugin.program.autowidget/resources/lib/refresh.py

```diff
@@ -269,54 +316,49 @@ def refresh_paths(notify=False, force=False):
     return True, "AutoWidget"
 
 
-def get_files_list(path, widget_id=None):
-    hash = utils.path2hash(path)
-    _, files, _ = utils.cache_expiry(hash, widget_id)
+def get_files_list(path, label=None, widget_id=None, background=True):

```

Comment:
We always do caching in the background, except in the case of exploding/cloning.

Reply:
I was thinking next page should not be done in the background but I guess if its always precached then this is no longer a problem